rocky/crisis_room/views.py

```diff
@@ -46,7 +52,11 @@ def get_list_for_org(self, organization: Organization) -> Union[List, None]:
             api_connector = OctopoesAPIConnector(settings.OCTOPOES_API, organization.code)
 
             return api_connector.list(self.ooi_types, valid_time=self.get_observed_at()).items
-        except RemoteException:
+        except ConnectorException as e:
+            messages.add_message(
+                self.request, messages.ERROR, _("Failed to get list of findings, check server logs for more details.")
+            )
+            logger.error("Failed to get list of findings for organization %s", organization.code, exc_info=e)
```

Comment:
You can also use `logger.exception`, it will give the same formatted error message without setting any flag (it sets `exc_info=True` by default which will print the stack traceback from the latest exception)